earthly/docs/Earthfile

```diff
@@ -48,39 +49,50 @@ MKDOCS_BUILD:
     COPY +mkdocs-material/services/docker-compose.kroki.yml .
 
     # Copy the src
+    RUN mkdir /extras 
     COPY --dir $src /docs
+    FOR --sep=", " file IN $extra_files
```

Comment:
We should standardize on space-separated for inputs that need multiple values. This is already the standard in other places, and it's also the default for bash (i.e., you can remove the `--sep` flag).